schema/partner_artist.js

```diff
@@ -0,0 +1,72 @@
+import gravity from '../lib/loaders/gravity';
+import Partner from './partner';
+import Artist from './artist/index';
+import numeral from './fields/numeral';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from 'graphql';
+
+const PartnerArtistType = new GraphQLObjectType({
+  name: 'PartnerArtist',
+  fields: () => ({
+
+    counts: {
+      type: new GraphQLObjectType({
+        name: 'PartnerArtistCounts',
+        fields: {
+          artworks: numeral(({ published_artworks_count }) =>
+            published_artworks_count),
+          for_sale_artworks: numeral(({ published_for_sale_artworks_count }) =>
+            published_for_sale_artworks_count),
+        },
+      }),
+      resolve: (partner_artist) => partner_artist,
+    },
+    id: {
+      type: GraphQLString,
+    },
+    is_display_on_partner_profile: {
+      type: GraphQLBoolean,
```

Comment:
huh? this isn't on the partner profile page. all we want is the bio. there
is a 'use default biography' flag which is specific to the partner profile
page (id the partner has supplied a bio but for some reason ticked the box
to display our bio on their profile instead of their own. apparently this
is specific to that page and doesn't apply to the artist page.

On Fri, Jun 10, 2016 at 6:12 PM, mzikherman <notifications@github.com>
wrote:

> In schema/partner_artist.js
> <https://github.com/artsy/metaphysics/pull/311#discussion_r66640171>:
>
> > +      type: new GraphQLObjectType({
> > +        name: 'PartnerArtistCounts',
> > +        fields: {
> > +          artworks: numeral(({ published_artworks_count }) =>
> > +            published_artworks_count),
> > +          for_sale_artworks: numeral(({ published_for_sale_artworks_count }) =>
> > +            published_for_sale_artworks_count),
> > +        },
> > +      }),
> > +      resolve: (partner_artist) => partner_artist,
> > +    },
> > +    id: {
> > +      type: GraphQLString,
> > +    },
> > +    is_display_on_partner_profile: {
> > +      type: GraphQLBoolean,
>
> Don't these need a resolve with display_on_partner_profile? That's the
> JSON that comes back from gravity.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artsy/metaphysics/pull/311/files/efa6da403c6869ecd3fcf79a6c796d2d181c6920#r66640171>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADBlTjjZsFCMdSdqJk7G82eNZbb97RRWks5qKYztgaJpZM4IzFFy>
> .
>



-- 

[A.]


Laura Brown

Artsy <http://artsy.net/>

laura@artsy.net <caroline@artsy.net>